### ISSUE TITLE
Editing export rules.

### ISSUE THREAD

**Issue by Cactucs** [avocode/avocode#566]
IMPORTANT: Please try to describe how did it happen.

While editing export rules.

There is a popup, but everything seems to work OK.

-----
App v.2.4.5
Source UID: 62957
```
TypeError: menu.sendActionToFirstResponder is not a function
  at HTMLDocument.<anonymous> (/home/vojta/bin/Avocode-linux-x64/resources/app.asar/src/window-event-handler.js:170:20)
  at HTMLDocument.handler (/home/vojta/bin/Avocode-linux-x64/resources/app.asar/src/space-pen-extensions.js:115:32)
  at HTMLDocument.jQuery.event.dispatch (/home/vojta/bin/Avocode-linux-x64/resources/app.asar/node_modules/space-pen/vendor/jquery.js:4681:9)
  at HTMLDocument.elemData.handle (/home/vojta/bin/Avocode-linux-x64/resources/app.asar/node_modules/space-pen/vendor/jquery.js:4359:46)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/home/vojta/bin/Avocode-linux-x64/resources/app.asar/src/command-registry.js:251:29)
  at /home/vojta/bin/Avocode-linux-x64/resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/home/vojta/bin/Avocode-linux-x64/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:520:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/home/vojta/bin/Avocode-linux-x64/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:355:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/home/vojta/bin/Avocode-linux-x64/resources/app.asar/src/window-event-handler.js:184:27)

```

**Comment by jankuca** [comment]
Hello, thanks for the report! Is it possible you tried to select the contents of a text field in the dialog or the export sidebar panel using Ctrl+A or a similar key stroke?

**Comment by Cactucs** [comment]
Yes

**Comment by jankuca** [comment]
Ok, thank you. This is a known bug happening in an old version of Electron (the engine on which Avocode runs) and old Atom (the framework we currently still use). We will be releasing a fixed version in the coming weeks.

**Comment by rychlis** [comment]
This is a long-running bug coming from the codebase of Atom upon which Avocode is built. **Please select the contents using the mouse cursor for now.** We are working on removing the Atom codebase from our app and have this done with the next major release.

**Comment by rychlis** [comment]
This was fixed in Avocode 2.8
